src/corelibs/U2Gui/src/ObjectViewModel.cpp

```diff
@@ -681,4 +681,17 @@ void GObjectViewAction::addToMenuWithOrder(QMenu *menu) {
     menu->addAction(this);
 }
 
+GObjectViewObjectHandler::~GObjectViewObjectHandler() {
```

Comment:
Remove as far as it's empty

Reply:
Default descructors are not virtual and here is virtual one. I have not changed this code, just moved it from .h to .cpp (our code style agreement)

Reply:
In this case remove it and use the `default` keyword like `virtual ~GObjectViewObjectHandler() = default;`

Reply:
Good idea. Fixed